help/md/dnInverse.md

```diff
@@ -0,0 +1,36 @@
+## name
+dnInverse
+
+## title
+Inverse distribution 
+
+## description
+`dnInverse()` inverts a probability distribution.
+
+`dnInverse(x).probability()` returns `1 / x.probability()`; 
+`dnInverse(x).lnProbability()` returns `-x.lnProbability()`.
+
+This provides a way to perform inference using conditional probabilities,
+for example where 
+Pr(x | Model, Condition) = Pr(x | Model) / Pr(x satisfies Condition)
```

Comment:
I'm not sure how you'd compute `Pr(x > 5)` using RevBayes; I'm only familiar with using `.clamp()` to get `Pr(x = 5)`.  I've created an example on this basis – which is slightly contrived but hopefully simple to understand.

The "real life" use case is with dnPhyloCTMC objects where likelihoods are not so trivial to compute.  A real-life example would be conditioning on observing variable characters, where the likelihood would be `Pr(observed characters | Characters are variable)`, and the latter could be computed by evaluating the probability of an invariant character.  (Of course this particular case is already implemented internally in RevBayes.)

Reply:
Thank you for updating the documentation! I can see now how `dnInverse()` can be used to calculate conditional probabilities.

I'm still wondering if the example is perhaps less than ideal, because one could achieve the same result with a simpler script that doesn't rely on `dnInverse`, simply by changing:

```
PrXandY(1, 1) * inv_y.probability()
```

in the last line to:

```
PrXandY(1, 1) / y.probability()```

If I understand correctly, that's what you point out yourself in the immediately preceding comment.

I understand that this distribution bridges a real feature gap that you've encountered in your research, so I don't want to be difficult here. Maybe it's perfectly okay for the example to just demonstrate _how_ to use the distribution, without showing _when_ it should be used. I definitely won't block the merge over this, and will leave it to your discretion if further changes need to be made.

Reply:
Thanks, I'm glad the example is improved; though as you say, the `dnInverse` function is redundant in this specific situation. 
@bredelings has helped me to formulate some text that puts across the use case a little more convincingly, which I've just added to an expanded description section.

As our actual use case relies on some custom nexus files I don't know that it's particularly well suited for the example section, but once this is in the public domain I will revisit the documentation page to see whether a self-contained example that more clearly expresses the motivation can be constructed.